test/graphs/test_mirror_dist_local_csr.cpp

```diff
@@ -0,0 +1,142 @@
+    if (graph.find(i) == graph.end()) {
+      graph[i] = std::vector<std::uint64_t>();
+    }
+  }
+  file.close();
+}
+
+template <typename T>
+bool isInVector(T element, pando::GlobalRef<pando::Vector<T>> vec) {
+  for (std::uint64_t i = 0; i < lift(vec, size); i++) {
+    if (fmap(vec, get, i) == element) {
+      return true;
+    }
+  }
+  return false;
+}
+
+class MirrorDLCSRInitEdgeList
+    : public ::testing::TestWithParam<std::tuple<const char*, std::uint64_t>> {};
+TEST_P(MirrorDLCSRInitEdgeList, MapExchange) {
+  using ET = galois::ELEdge;
+  using VT = galois::ELVertex;
+  using Graph = galois::MirrorDistLocalCSR<VT, ET>;
+  galois::HostLocalStorageHeap::HeapInit();
+
+  const std::string elFile = std::get<0>(GetParam());
+  const std::uint64_t numVertices = std::get<1>(GetParam());
+
+  pando::Array<char> filename;
+  EXPECT_EQ(pando::Status::Success, filename.initialize(elFile.size()));
+  for (uint64_t i = 0; i < elFile.size(); i++)
+    filename[i] = elFile[i];
+
+  Graph graph =
+      galois::initializeELDLCSR<Graph, galois::ELVertex, galois::ELEdge>(filename, numVertices);
+
+  auto dims = pando::getPlaceDims();
+  for (std::int16_t nodeId = 0; nodeId < dims.node.id; nodeId++) {
+    pando::GlobalRef<pando::Array<Graph::MirrorToMasterMap>> localMirrorToRemoteMasterOrderedMap =
+        graph.getLocalMirrorToRemoteMasterOrderedMap(nodeId);
+    for (std::uint64_t i = 0ul; i < lift(localMirrorToRemoteMasterOrderedMap, size); i++) {
+      Graph::MirrorToMasterMap m = fmap(localMirrorToRemoteMasterOrderedMap, get, i);
+      Graph::VertexTopologyID masterToopologyID = m.getMaster();
+      Graph::VertexTokenID masterTokenID = graph.getTokenID(masterToopologyID);
+      std::uint64_t physicalHost = graph.getPhysicalHostID(masterTokenID);
+      pando::GlobalRef<pando::Vector<pando::Vector<Graph::MirrorToMasterMap>>>
+          localMasterToRemoteMirrorMap = graph.getLocalMasterToRemoteMirrorMap(physicalHost);
+      pando::GlobalRef<pando::Vector<Graph::MirrorToMasterMap>> mapVectorFromHost =
+          fmap(localMasterToRemoteMirrorMap, get, nodeId);
+      EXPECT_TRUE(isInVector<Graph::MirrorToMasterMap>(m, mapVectorFromHost));
+    }
+  }
+  graph.deinitialize();
+}
+
+INSTANTIATE_TEST_SUITE_P(
+    SmallFiles, MirrorDLCSRInitEdgeList,
+    ::testing::Values(std::make_tuple("/pando/graphs/simple.el", 10),
+                      std::make_tuple("/pando/graphs/rmat_571919_seed1_scale10_nV1024_nE10447.el",
+                                      1024)));
+
+INSTANTIATE_TEST_SUITE_P(
+    DISABLED_BigFiles, MirrorDLCSRInitEdgeList,
+    ::testing::Values(
+        std::make_tuple("/pando/graphs/rmat_571919_seed1_scale11_nV2048_nE22601.el", 2048),
+        std::make_tuple("/pando/graphs/rmat_571919_seed1_scale12_nV4096_nE48335.el", 4096),
+        std::make_tuple("/pando/graphs/rmat_571919_seed1_scale13_nV8192_nE102016.el", 8192),
+        std::make_tuple("/pando/graphs/rmat_571919_seed1_scale14_nV16384_nE213350.el", 16384)));
```

Comment:
when you delete DISABLED do these pass?

Reply:
Yes, but it takes a bit longer. Do you want me to enable them?